### DISCUSSION TITLE
Multiple projects and auth

### DISCUSSION THREAD

**Discussion by danieleorlando** [pocketbase/pocketbase#1338] [Q&A] [answered]
How can I handle multiple projects with one single user auth?

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand the question. PocketBase doesn't have "apps/projects" separation.
If you want to maintain multiple apps/projects from one db than the only way will be to prefix your collections (eg. `app1_tasks`, `app2_tasks`, etc.).
But in general I wouldn't recommend this. It is better to isolate each of your apps/projects in their own `pb_data` directory (this way it will be also easier later to move one app/project to another server). You can start one executable multiple times with different port and dir flag, eg:

```sh
./pocketbase serve --dir="/path/to/pb_data1" --http="127.0.0.1:8091"
./pocketbase serve --dir="/path/to/pb_data2" --http="127.0.0.1:8092"
```

**Comment by danieleorlando** [comment, marked as answer]
Hi @ganigeorgiev .
sorry for the lack of clarity. I know there's no separation for projects.
I meant multiple separated instances of pocketbase.

In my scenario, I have multiple apps but I need only one login for all of them

**Reply by ganigeorgiev** [reply]
>  I have multiple apps but I need only one login for all of them

No, this is not possible, _at least not out of the box_.

One workaround could be to create a middleware service that will receive the auth credentials and then will check in each app whether the user is registered. But this is too use case specific and there are a lot of different ways to handle it and I'm not sure what to recommend. It depends whether you want to have relations between the different applications, how the users are expected to interact with each individual app and api, whether it is ok to have duplicated/synced user data, etc.

**Reply by danieleorlando** [reply]
Thanks for clearing that up
